### PR TITLE
Open a project

### PR DESCRIPTION
**Pull request by mbr4477**
Also fixes chunk card and view takes issue of displaying chunk label instead of "Verse <number>"

### CODE REVIEW

File: src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectpage/model/ProjectPageModel.kt [hunk 1]
```diff
@@ -64,28 +65,23 @@ class ProjectPageModel {
...
-        projectTitle = newProject.titleKey
-        projectPageActions
-                .getChildren(newProject)
-                .observeOnFx()
-                .subscribe { childCollections ->
-                    // Now we have the children of the project collection
-                    children.clear()
-                    children.addAll(childCollections)
-                }
+    fun initializeView() {
```
Comment (jsarabia) [inline src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectpage/model/ProjectPageModel.kt]: initialize what view?

Comment (mbr4477) [reply src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectpage/model/ProjectPageModel.kt]: The view bound to this model (through the viewmodel), the project editor page.

Comment (mbr4477) [reply src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectpage/model/ProjectPageModel.kt]: Could be renamed to just initializeData

File: src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ChunkCard.kt [hunk 2]
```diff
@@ -17,7 +17,7 @@ class ChunkCard(val chunk: Chunk) : VBox() {
             alignment = Pos.CENTER
             spacing = 10.0
             // TODO: Localization
-            label(chunk.labelKey) {
+            label("Verse ${chunk.start}") {
```
Comment (jsarabia) [inline src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ChunkCard.kt]: the label field of content is meant to be the key for the localized word for "verse"

Comment (mbr4477) [reply src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ChunkCard.kt]: So this should be `"${messages[chunk.labelKey]} ${chunk.start}"`? And then add `verse=Verse` in the Messages_en.properties.

**Review by jsarabia — APPROVED** [review]

**Pull request merged**